libsolidity/ast/Types.cpp

```diff
@@ -110,6 +110,15 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 	return transformed;
 }
 
+std::string typesToString(TypePointers const& _types, bool _short) {
+	std::string str;
+	str += '(';
+	for (auto it = _types.begin(); it != _types.end(); ++it)
+		str += (*it)->toString(_short) + (it + 1 == _types.end() ? "" : ",");
+	str += ")";
+	return str;
+}
```

Comment:
```suggestion
string typesToString(TypePointers const& _types, bool _short)
{
	string str;
	str += '(';
	for (auto it = _types.begin(); it != _types.end(); ++it)
		str += (*it)->toString(_short) + (it + 1 == _types.end() ? "" : ",");
	str += ")";
	return str;
}
```

as some minor coding style remarks...
We could probably also simplify further to something like

```suggestion
string typesToString(TypePointers const& _types, bool _short)
{
  return '(' + joinHumanReadable(_types | ranges::views::transform([&](auto const* _type) { return _type->toString(_short); }) + ')';
}
```

Reply:
(For reference: https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md describes most of the general coding style we try to adhere to)